app/src/main/res/values/servers.xml

```diff
@@ -32,7 +37,7 @@
     </string-array>
 
     <string-array name="content_server_account_create_urls_array">
-        <item>https://content.bibletranslationtools.org/user/sign_up</item>
+        <item>https://wacs.bibletranslationtools.org/user/sign_up</item>

```

Comment:
Same here -- please use `content.bibletranslationtools.org`, not `wacs...`.

Reply:
Done.